### PR TITLE
Updating the Cython prerequisite to v0.19 in install.rst 

### PR DESCRIPTION
**Pull request by kvyh**
resolving issue #4705 by changing the minimum version for building from v0.15 to v0.19 in docs/install.rst.

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -434,4 +434,4 @@ installing astropy) is to run this from the root of the source tree::
 
 There are also alternative methods of :ref:`running-tests`.
 
-.. include:: development/workflow/known_projects.inc
\ No newline at end of file
+.. include:: development/workflow/known_projects.inc
```
Comment (pllim) [inline docs/install.rst]: I don't understand why there is a diff here. Tab/space confusion? Trailing space?

File: CHANGES.rst [hunk 2]
```diff
@@ -304,6 +304,8 @@ Other Changes and Additions
 
 - Reduce Astropy's import time (``import astropy``) by almost a factor 2. [#4649]
 
+- Cython prerequisite for building changed to v0.19 in install.rst
```
Comment (pllim) [inline CHANGES.rst]: Add issue number. See other entries for formatting.

**Comment by pllim** [comment]
Nitpicking -- squash the commits into one.

File: docs/install.rst (hunk 1 above)
Comment (kvyh) [inline docs/install.rst]: I don't know, I didn't modify that part of the document at all.

Comment (eteq) [inline docs/install.rst]: It's a trailing space - your editor probably did it automatically.

**Comment by eteq** [comment]
While #4710 did most of this, this PR correctly caught that there should be a changelog mention, too.  So I'll merge this also, to add in the changelog entry.  (Will also use the new auto-squashing feature to satisfy @pllim!)

**Comment by eteq** [comment]
Hmm, interesting: the squash like that apparently creates a single commit on master rather than squashing the PR and then making a merge commit.  That is very bad for the problem of maintaining bugfix branches...  Fortunately this PR is for 1.2, so it's a non-issue.

**Comment by MSeifert04** [comment]
@astrofrog Is the bot message intentional here? I wouldn't think it should comment on already merged PRs.

**Comment by astrofrog** [comment]
@MSeifert04 - yep, it's a bug! Thanks for reporting :)

**Pull request merged**